security/calico-network-policy.md

```diff
@@ -97,6 +97,12 @@ The Kubernetes network policy specification defines the following behavior:
 
 For compatibility with Kubernetes, **Calico network policy** follows the same behavior for Kubernetes pods.  For other endpoint types (VMs, host interfaces), Calico network policy is default deny. That is, only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
 
+### Before you begin
+
+Calicoctl must be **installed** and **configured** before use. If calicoctl is printing *"no etcd endpoints specified"*, please configure Calico to either a Kubernetes or etcd datastore.
```

Comment:
I would suggest we change some of the language here. Something along the lines of.

Calicoctl must be installed and configured before use. For example, by default calicoctl will use etcd as the datastore, but many Calico installation manifests configure Kubernetes as the datastore. You can find more information on how to install and configure calicoctl in the following links: